src/local_sensitivity/derivative_wrappers.jl

```diff
@@ -201,7 +201,11 @@ end
 function _vecjacobian!(dλ, λ, p, t, S::SensitivityFunction, isautojacvec::ZygoteVJP, dgrad, dy)
   @unpack y, sensealg = S
   prob = getprob(S)
-  f = prob.f
+  if prob isa Union{SDEProblem}
+    @unpack f = S
```

Comment:
the goal should probably be to make the other SensitivityFunctions all have the `f` in there, but this works for now.

Reply:
We could merge without this, but open an issue on it.